packages/docs/concepts/blocks.yaml

```diff
@@ -64,7 +64,8 @@ _ref:
             - `blocks: array`: A array of blocks to render to the default `content` area for `container`, `context` and `list` blocks. See [layout](/layout) for more details on how to use the `blocks` array.
             - `events: object`: Used to defined [`actions`](/events-and-actions) that run when the block triggers an [`event`](/events-and-actions).
             - `layout: object`: Used to define the [layout](/layout) properties for a block. __Operators are evaluated__.
-            - `loading: object`: Used to overwrite a block's default loading behavior.
+            - `skeleton: object`: Used to overwrite a block's default loading behavior by rendering a skeleton. Any block types from [`@lowdefy/blocks-basic`](https://www.npmjs.com/package/@lowdefy/blocks-basic) and [`@lowdefy/blocks-loaders`](https://www.npmjs.com/package/@lowdefy/blocks-loaders) can be used in the skeleton definition, including sub-blocks.
+            - `loading: boolean`: When true, renders the block's in loading. If a `skeleton` is defined, it will be rendered else, the block's default loading behavior will be rendered.
```

Comment:
When true, renders the block in loading. If a `skeleton` is defined, it will be rendered, else the block's default loading behavior will be rendered.